R/pkg/R/SQLContext.R

```diff
@@ -434,6 +388,7 @@ read.orc <- function(path, ...) {
 #' Loads a Parquet file, returning the result as a SparkDataFrame.
 #'
 #' @param path path of file to read. A vector of multiple paths is allowed.
+#' @param ... additional external data source specific named properties.
```

Comment:
@felixcheung I got this CRAN doc error on the last run:

```
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... WARNING
Undocumented arguments in documentation object 'read.parquet'
  '...'
```

I guess this is what it wants, but not sure why it didn't come up before?

Reply:
Oops, I missed that, sorry. I'll incorporate both changes.